README_GAMELIFT.md

```diff
@@ -2,39 +2,110 @@
+    copy D:\github\o3de-multiplayersample\build\windows_mono\bin\profile\*.* %1
+    
+    rem Copy the AWSCore files
+    copy D:\github\o3de-multiplayersample\build\windows_mono\bin\profile\Gems\AWSCore\*.* %1\Gems\AWSCore
+    
+    rem Copy the launch_client / launch_server files
+    copy D:\github\o3de-multiplayersample\launch_*.* %1
+    ```
 
-### Testing
 
-1. Open a command terminal from within the `GameLiftLocal` directory you downloaded before, and run it with the following command:
-    ```sh
-    java -jar GameLiftLocal.jar
-    ```
+## Prepare for GameLift
+### Upload the build to GameLift
+Note: Builds are tied to Fleets; you may want to delete the existing build and fleet via the AWS Gamelift dashboard just so you don't accidentally reference old builds or old fleets in future steps. 
+`
+aws gamelift upload-build --operating-system WINDOWS_2012 --build-root .\GameLiftPackageWindows\ --name MultiplayerSample --build-version v1.0 --region us-west-2
+`
+This returns the build id which will be used in the next steps. Example Build ID: build-f447776f-b4ba-4f1e-99cf-09d49f114942
 
-1. From another terminal within the root of the MultiplayerSample project, start the server launcher (assumes the executable is the result of a `profile` build):
-    ```sh
-    ./build/windows/bin/profile/MultiplayerSample.ServerLauncher.exe --sv_gameLiftEnabled=true --sv_dedicated_host_onstartup=false --loadlevel=NewStarbase
-    ```
-Note: You may be inclined to move these cvars into a cfg file and start the game by passing the --console-command-file parameter, but don't. Some cvars are used during a system component Activate(), but the console-command-file is executed after all system components have been activated. (example: sv_gameLiftEnabled is used inside AWSGameLiftServerSystemComponent::Activate()).
+### Create Fleet
+After running this command it'll take about an hour for the fleet to activate. Check the status on the GameLift dashboard. 
 
-1. Create a game session with the below command:
-    ```sh
-    aws gamelift create-game-session --endpoint-url http://localhost:8080 --maximum-player-session-count 2 --fleet-id fleet-123 --game-session-id hello-mps
-    ```
-    You should observe logs in the `GameLiftLocal` terminal which indicate it handled the create-game-session request, and see the `ServerLauncher` load its level.
+`
+aws gamelift create-fleet --region us-west-2 --name GlLyTest2F --ec2-instance-type c5.large --fleet-type ON_DEMAND --build-id build-f447776f-b4ba-4f1e-99cf-09d49f114942 --runtime-configuration "GameSessionActivationTimeoutSeconds=300, MaxConcurrentGameSessionActivations=2, ServerProcesses=[{LaunchPath=C:\game\MultiplayerSample.ServerLauncher.exe, Parameters= --sv_gameLiftEnabled=true --sv_dedicated_host_onstartup=false --loadlevel=NewStarbase, ConcurrentExecutions=10}]" --ec2-inbound-permissions "FromPort=33435,ToPort=33435,IpRange=0.0.0.0/0,Protocol=UDP" "FromPort=33235,ToPort=33235,IpRange=0.0.0.0/0,Protocol=UDP"
```

Comment:
```suggestion
aws gamelift create-fleet --region us-west-2 --name GlLyTest2F --ec2-instance-type c5.large --fleet-type ON_DEMAND --build-id build-f447776f-b4ba-4f1e-99cf-09d49f114942 --runtime-configuration "GameSessionActivationTimeoutSeconds=300, MaxConcurrentGameSessionActivations=1, ServerProcesses=[{LaunchPath=C:\game\MultiplayerSample.ServerLauncher.exe, Parameters= --sv_gameLiftEnabled=true --sv_dedicated_host_onstartup=false --loadlevel=NewStarbase, ConcurrentExecutions=1}]" --ec2-inbound-permissions "FromPort=33435,ToPort=33435,IpRange=0.0.0.0/0,Protocol=UDP" "FromPort=33235,ToPort=33235,IpRange=0.0.0.0/0,Protocol=UDP"
```

Unclear why you are opening two UDP ports and why you are launching 10 processes

Reply:
Fixed! This burnt me as well, readme started as a copy from old documentation. I didn't have the correct ports open so the client could never connect.